data/devices/xiaomi.yml

```diff
@@ -42,6 +42,44 @@
     default_brightness: 162
     theme: portrait_hdpi
 
+- name: Xiaomi Mi4C
+  id: libra
+  codenames:
+    - libra
+  architecture: arme64-v8a
```

Comment:
The build failed because there's a typo here. It should be `arm64-v8a`, not `arme64-v8a`.